discrete_summary_checker/data/CruiseInformation.csv

```diff
@@ -0,0 +1,246 @@
+AR42-02,Neil Armstrong,2020-02-24T12:00:00,2020-03-04T23:59:00,Pioneer glider cp_340 recovery
+TS20200301,Tenacious II,2020-03-01T20:00:00,2020-03-02T15:00:00,Pioneer 13b glider deployments
+WJ20200323,Warren Jr.,2020-03-23T00:00:00,2020-03-27T23:59:00,Pioneer 13c AUV deployments
+AJ20200506,Adolf Jensen,2020-05-06T12:00:00,2020-05-08T12:00:00,Irminger glider gi_525 recovery
+TS20200522,Tenacious II,2020-05-22T06:00:00,2020-05-22T23:59:59,Pioneer 13b glider recoveries
+ZP20200604,Zephyr,2020-06-04T08:00:00,2020-06-08T23:59:59,Papa Glider gp_537 and gp_575 recoveries
+AR44,Neil Armstrong,2020-06-07T16:26:00,2020-06-16T16:32:00,Pioneer 14
+EK20200612,Elakha,2020-06-12T16:06:00,2020-06-12T23:00:00,Endurance Glider deployment 320_00006
+TS20200613,Tenacious II,2020-06-13T11:00:00,2020-06-14T04:00:00,Pioneer 14a Glider deployments
+TS20200629,Tenacious II,2020-06-29T17:00:00,2020-06-30T09:00:00,Pioneer glider cp_380 recovery
+TN380,Thomas G. Thompson,2020-07-04T00:00:00,2020-07-16T19:00:00,Endurance 13 moorings profilers and gliders
+MZ20200731,Mattanza,2020-07-31T03:00:00,2020-07-31-T23:59:59,Pioneer gliders cp_339 and cp_583 recoveries
+TN382,Thomas G. Thompson,2020-08-02T00:00:00,2020-08-28T23:59:59,Cabled (rvdata)
+AR46,Neil Armstrong,2020-08-08T12:18:00,2020-09-04T17:02:00,Irminger 7
+EK20200814,Elakha,2020-08-14T14:15:00,2020-08-14T18:50:00,Endurance CSPP deployment CE02SHSP-00018
+TS20200819,Tenacious II,2020-08-19T07:30:00,2020-08-20T06:30:00,Pioneer 14b Glider deployments and recoveries
+PS20200917,Pacific Storm,2020-09-17T17:30:00,2020-09-20T17:30:00,Endurance CSPP and glider operations
+TS20200925,Tenacious II,2020-09-25T21:00:00,2020-09-26T15:00:00,Pioneer 14c Glider deployments
+ZP20200930,Zephyr,2020-09-30T17:00:00,2020-10-03T23:59:59,Papa 7a Glider deployments
+EN657,Endeavor,2020-10-13T13:00:00,2020-10-18T16:00:00,Pioneer glider cp_340 recovery
+AR48,Neil Armstrong,2020-10-28T13:34:00,2020-11-11T16:36:00,Pioneer 15
+PS20201120,Pacific Storm,2020-11-20T16:30:00,2020-11-20T23:30:00,Endurance glider deployments
+TS20201121,Tenacious II,2020-11-21T09:00:00,2020-11-22T06:00:00,Pioneer 15a glider deployments
+SI20201212,Scarlett Isabella,2020-12-12T01:00:00,2020-12-12T21:30:00,Pioneer PMUI-15 recovery
+EK20210120,Elakha,2021-01-20T16:40:00,2021-01-20T23:50:00,Endurance glider deployments
+TS20210131,Tenacious II,2021-01-31T08:00:00,2021-02-01T03:00:00,Pioneer 15b glider deployments
+WJ20210209,Warren Jr.,2021-02-09T22:00:00,2021-02-11T13:00:00,Pioneer glider cp_340 recovery
+EK20210210,Elakha,2021-02-10T17:15:00,2021-02-11T01:00:00,Endurance glider recoveries
+SKQ202104S,Sikuliaq,2021-03-24T16:10:00,2021-04-07T18:00:00,Endurance 14 moorings profilers and gliders
+AR52,Neil Armstrong,2021-03-29T20:19:00,2021-04-17T11:37:00,Pioneer 16
+TS20210419,Tenacious II,2021-04-19T05:00:00,2021-04-20T04:00:00,Pioneer 16a glider recoveries
+EK20210419,Elakha,2021-04-19T14:50:00,2021-04-19T20:38:00,Endurance CE01ISSP-00017 and CE05MOAS-GL383-00010
```

Comment:
Looks good.  But this will need to be updated with the more recent version at somepoint.

Reply:
Yeah, it is static but should be easy to update by just copy-pasting.